server/db/recreate-schema.sql

```diff
@@ -1,100 +1,154 @@
 -- Drop tables in case they already exist
 -- Drop tables in case they already exist
-DROP TABLE if exists users;
+DROP TABLE IF exists users;
 DROP TABLE IF EXISTS blocks CASCADE;
 DROP TABLE IF EXISTS topics CASCADE;
-DROP TABLE IF EXISTS block_topics CASCADE;
 DROP TABLE IF EXISTS questions CASCADE;
 DROP TABLE IF EXISTS volunteers CASCADE;
-DROP TABLE IF EXISTS volunteer_topic CASCADE;
-DROP TABLE IF exists volunteer_answers CASCADE;
+DROP TABLE IF EXISTS volunteer_blocks CASCADE;
+DROP TABLE IF EXISTS volunteer_answers CASCADE;
 
 -- Create tables
-CREATE TABLE users (
-  id SERIAL PRIMARY KEY,
-  email    VARCHAR(200) NOT NULL,
-  password VARCHAR(200) NOT NULL
-);
+--  users definition
 
+-- Drop table
 
+-- DROP TABLE  users;
 
-CREATE TABLE blocks(
-  block_id INT GENERATED ALWAYS AS IDENTITY,
-   block_name VARCHAR(255) NOT NULL,
-   PRIMARY KEY( block_id)
+CREATE TABLE  users (
+	id serial NOT NULL,
+	email varchar(200) NOT NULL,
+	"password" varchar(200) NOT NULL,
+	CONSTRAINT users_pkey PRIMARY KEY (id)
 );
 
+-- Permissions
+
+-- ALTER TABLE  users OWNER TO app_user;
+-- GRANT ALL ON TABLE  users TO app_user;
+
 
+-----------------------------------------------------------------------------
+--  blocks definition
 
+-- Drop table
 
-CREATE TABLE topics(
-   topic_id INT GENERATED ALWAYS AS IDENTITY,
-   topic_name VARCHAR(255) NOT NULL,
-   document_name VARCHAR(255) ,
-   document_link VARCHAR(255) ,
-   PRIMARY KEY(topic_id)
+-- DROP TABLE  blocks;
+
+CREATE TABLE  blocks (
+	block_id int4 NOT NULL GENERATED ALWAYS AS IDENTITY,
+	block_name varchar(255) NOT NULL,
+	CONSTRAINT blocks_pkey PRIMARY KEY (block_id)
 );
-CREATE TABLE block_topics(
```

Comment:
You have removed the many-to-many relationship between blocks and topics, as you said was not what was wanted. Assuming this is what you now want to happen, so one to many between blocks and topics then there's no need to make your other changes.  I am not saying you should not do this, just making you aware this has changed the previously agree functionality.

Reply:
I did not get your point ?

Reply:
You have changed the functionality of the system with this schema change. Previously there was a many-to-many relationship between the Blocks and Topics and you have changed that to One-to-Many. So a Topic can only be in one Block. I just need you to acknowledge this is an intentional change.  

As a side effect of this change changing to to volunteer_blocks rather than volunteer_topics is also slightly pointless as you know the block the topic is from.

Reply:
Yes the change in functionality is valid here in our case, since their is no point to have a direct many to many relationship between the topics and blocks.But we need a many to many relationship between the Volunteer Block and topics , and it's reflected in this new db design changes.

Know we can have a BLOCKs with multiple TOPICs ----  then we can have many VOLUNTEER_BLOCKS  related to a BLOCK --- in the end we will have multiple TOPICS with multiple BLOCKS in different VOLUNTEER_BLOCKS

I believe this design is reflecting the business better than the previous one.

You have this sentence " I just need you to acknowledge this is an intentional change", but I did not get your point of this sentence , what do you mean exactly ?